### PR TITLE
Migrate custom detail page documentation (designers guide)

### PR DESCRIPTION
**Pull request by philipgatzka**

### CODE REVIEW

File: source/designers-guide/custom-detail-page/index.md [hunk 1]
```diff
@@ -486,3 +486,17 @@ The code to remove the image zoom is pretty simple. We check if the main `conten
 If you want to take a closer look at all the code we have written in this tutorial you can download this plugin:
 
 <strong><a href="{{ site.url }}/exampleplugins/SwagCustomDetailTheme.zip">Plugin download</a></strong>
+
+The Theme is packed inside a plugin, as that's a requirement to distribute themes via our store.
```
Comment (mitelg) [inline source/designers-guide/custom-detail-page/index.md]: The theme is packed inside a plugin. This is a requirement to distribute themes via our store.

File: source/designers-guide/custom-detail-page/index.md [hunk 2]
```diff
@@ -486,3 +486,17 @@ The code to remove the image zoom is pretty simple. We check if the main `conten
...
+The structure of the Theme is exactly the same as before, it just sits a few layers deeper in the directory hierarchy:
```
Comment (mitelg) [inline source/designers-guide/custom-detail-page/index.md]: The structure of the theme is exactly the same as before, it is just placed a few layers deeper in the directory hierarchy:

File: source/designers-guide/custom-detail-page/index.md [hunk 3]
```diff
@@ -486,3 +486,17 @@ The code to remove the image zoom is pretty simple. We check if the main `conten
...
+
+```
+SwagCustomDetailTheme
+└── Resources
+    └── Themes
+        └── Frontend
+            └── *The theme folder*
+```
+
+In addition, the root directory of the plugin contains a plugin base file and a `plugin. xml` with meta information.
```
Comment (mitelg) [inline source/designers-guide/custom-detail-page/index.md]: plugin.xml

File: source/designers-guide/custom-detail-page/index.md (hunk 1 above)
Comment (philipgatzka) [reply source/designers-guide/custom-detail-page/index.md]: Done

File: source/designers-guide/custom-detail-page/index.md (hunk 2 above)
Comment (philipgatzka) [reply source/designers-guide/custom-detail-page/index.md]: Done

File: source/designers-guide/custom-detail-page/index.md (hunk 3 above)
Comment (philipgatzka) [reply source/designers-guide/custom-detail-page/index.md]: Done

**Pull request merged**